src/control/lib/control/pool.go

```diff
@@ -548,8 +548,8 @@ func (pqr *PoolQueryResp) UpdateState() error {
 		pqr.State = daos.PoolServiceStateUnknown
 	}
 
-	// Update the Pool state as Degraded, if initial state is Ready and any target is disabled
-	if pqr.State == daos.PoolServiceStateReady && pqr.DisabledTargets > 0 {
+	// Update the Pool state as Degraded if initial state is Ready and rebuild is busy.
+	if pqr.Rebuild != nil && pqr.Rebuild.State == daos.PoolRebuildStateBusy {
```

Comment:
I wonder, can we tell if the rebuild failed? I'd think if rebuild is running OR failed, the pool should be considered degraded.

Reply:
As an end user, when the rebuild failed I would like to have a different state indicating that some unexpected errors occurred.